src/app/layout/header/dialog-user/dialog-user.component.spec.ts

```diff
@@ -101,6 +101,7 @@ describe('DialogUserComponent', () => {
     fixture.detectChanges();
 
     birthdate.value = todayDate();
+    console.log(birthdate.value);
```

Comment:
I think you can remove this log

Reply:
yeah, forgot while debugging test. Thanks.